core/src/main/scala/org/apache/spark/api/python/SerDeUtil.scala

```diff
@@ -78,7 +78,7 @@ private[spark] object SerDeUtil extends Logging {
    * Choose batch size based on size of objects
    */
   private[spark] class AutoBatchedPickler(iter: Iterator[Any]) extends Iterator[Array[Byte]] {
-    private val pickle = new Pickler()
+    private val pickle = new Pickler(true, false)
```

Comment:
nit: I'd prefer to use named arguments for readability: `new Pickler(useMemo=true, valueCompare=false)`.

Reply:
Actually, there is a java way to do it `func(/*arg-name=*/ value)`

Reply:
fixed in https://github.com/apache/spark/pull/31682/commits/279f536167387ab21a56883f4d8827df9b03aef3, for some reason scalastyle doesn't like more than one of these in one line...

Reply:
cool, thanks!